examples/crontabs-crd.json

```diff
@@ -0,0 +1,20 @@
+{
+  "kind": "CustomResourceDefinition", 
+  "spec": {
+    "scope": "Namespaced", 
+    "version": "v1", 
+    "group": "stable.example.com", 
+    "names": {
+      "shortNames": [
+        "ct"
+      ], 
+      "kind": "CronTab", 
+      "plural": "crontabs", 
+      "singular": "crontab"
+    }
+  }, 
+  "apiVersion": "apiextensions.k8s.io/v1beta1", 
+  "metadata": {
+    "name": "crontabs.stable.example.com"
+  }
+}
```

Comment:
new line ☘️

Reply:
:)  I call it AAFN (Add A F****** Newline).

Reply:
😱